src/Microsoft.Health.Dicom.Core/Features/Common/DicomTagParser.cs

```diff
@@ -27,24 +28,79 @@ public bool TryParse(string dicomTagPath, out DicomTag[] dicomTags, bool support
                 return false;
             }
 
+            DicomTag dicomTag = GetTagFromKeywordOrNumber(dicomTagPath);
 
+            dicomTags = new DicomTag[] { dicomTag };
+            return dicomTag != null;
+        }
+
+        public bool TryParseToDicomItem(string dicomTagPath, out DicomItem dicomItem)
+        {
+            dicomItem = null;
+
+            if (string.IsNullOrWhiteSpace(dicomTagPath))
+            {
+                return false;
+            }
+
+            DicomTag[] dicomTags = ParseMultipleStandardTag(dicomTagPath);
+
+            if (dicomTags.Length == 1)
+            {
+                var tag = GetTagFromKeywordOrNumber(dicomTagPath);
+                if (tag != null) dicomItem = new DicomValuelessItem(tag);
+                return tag != null;
+            }
+
+            if (dicomTags.Length > 2)
+            {
+                throw new NotImplementedException(DicomCoreResource.NestedSequencesNotSupported);
```

Comment:
Is this tracked for 500 return code? We general wrap all exceptions with DicomValidationExceptions

Reply:
Yes this return 500. Changed to DicomValidationException